packages/web-react/src/types/drawer.ts

```diff
@@ -21,9 +22,12 @@ export type DrawerPanelHandlingProps = {
   closeOnEscapeKeyDown?: boolean;
 };
 
-export interface DrawerCloseButtonProps<C = void> extends Omit<SpiritButtonElementProps, 'children' | 'color'> {
+export interface DrawerCloseButtonProps<C = void, S = void>
+  extends Omit<SpiritButtonElementProps, 'children' | 'color' | 'size'> {
```

Comment:
Why is the size omitted and added manually?

Reply:
You should ask yourself: https://github.com/lmc-eu/spirit-design-system/pull/1987/files#diff-eb30718f9e6dc4f5c3f08d9d0463167d6eafbd43c3e8fbb08f2b861eaae8c8a7R25

Anyway, if it isn't here, the property type has probably been set from HTMLElement.
<img width="864" alt="image" src="https://github.com/user-attachments/assets/f6cf18a5-26b6-4704-b10f-4a7d507ba0d5" />